libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1492,6 +1493,24 @@ bool AP_InertialSensor::gyro_calibrated_ok_all() const
     return (get_gyro_count() > 0);
 }
 
+// Prearm check to verify that we have sane sched loop rates set based on Gyro backend rates
+bool AP_InertialSensor::pre_arm_check_gyro_backend_rate_hz(char* fail_msg, uint16_t fail_msg_len) const
+{
+#if AP_SCHEDULER_ENABLED
+    for (uint8_t i=0; i<get_gyro_count(); i++) {
+        if (!_use(i) || _backends[i] == nullptr) {
+            continue;
+        }
+        if (_backends[i]->get_gyro_backend_rate_hz() < (2*_loop_rate)) {
```

Comment:
```suggestion
        const float required_rate_mult = 1.8;
        if (_backends[i]->get_gyro_backend_rate_hz() < (2*_loop_rate)) {
```
... used in two places